packages/block-library/src/paragraph/edit.js

```diff
@@ -176,12 +128,33 @@ function ParagraphBlock( {
 
 	const richTextStyles = useMemo(
 		() => ( {
-			backgroundColor: backgroundColor.color,
-			color: textColor.color,
 			fontSize: fontSize.size ? fontSize.size + 'px' : undefined,
 			direction,
 		} ),
-		[ backgroundColor.color, textColor.color, fontSize.size, direction ]
+		[ fontSize.size, direction ]
+	);
+
+	const { TextColor, BackgroundColor, InspectorControlsColorPanel } = __experimentalUseColors(
+		[
+			{ name: 'textColor', property: 'color' },
+			{ name: 'backgroundColor', property: 'background-color', className: 'has-background' },
```

Comment:
`property` should be `backgroundColor` right, and it defaults to `name` here so it should be fine without this explicit `'background-color'`?